charts/backstage/values.yaml

```diff
@@ -81,26 +81,38 @@ upstream:
         cpu: 1000m
         memory: 2.5Gi
         ephemeral-storage: 5Gi
+    startupProbe:
+      # This gives enough time upon container startup before the liveness and readiness probes are triggered.
+      # Giving (120s = initialDelaySeconds + failureThreshold * periodSeconds) to account for the worst case scenario.
+      httpGet:
+        path: /.backstage/health/v1/liveness
+        port: backend
+        scheme: HTTP
+      initialDelaySeconds: 30
+      timeoutSeconds: 4
+      periodSeconds: 20
+      successThreshold: 1
+      failureThreshold: 3
     readinessProbe:
       failureThreshold: 3
       httpGet:
-        path: /healthcheck
-        port: 7007
+        path: /.backstage/health/v1/readiness
+        port: backend
         scheme: HTTP
-      initialDelaySeconds: 30
+      # initialDelaySeconds: 30
       periodSeconds: 10
       successThreshold: 2
-      timeoutSeconds: 2
+      timeoutSeconds: 4
```

Comment:
could this be even higher for slow clusters? why not 10s?

Reply:
See discussion in https://github.com/redhat-developer/rhdh-chart/pull/50#discussion_r1883987372